PlaygroundAppUITests/XCUIElement+AsyncTests.swift

```diff
@@ -0,0 +1,42 @@
+import XCUITestHelpers
+import XCTest
+
+class XCUIElementAsyncTests: XCTestCase {
+
+    let app = XCUIApplication()
+
+    override func setUpWithError() throws {
+        continueAfterFailure = false
+        app.launch()
+    }
+
+    func testWaitsForPredicateOnElement() {
+        XCTAssertEqual(
+            app.staticTexts["a"].waitFor(
+                predicate: NSPredicate(format: "isEnabled == true"),
+                timeout: 2 // Interestingly, on Xcode 13.1, a 1 second timeout results in a failure
+            ),
+            .completed
+        )
+        // Test the opposite scenario must also be true.
+        XCTAssertEqual(
+            app.staticTexts["a"].waitFor(
+                predicate: NSPredicate(format: "isEnabled == false"),
+                timeout: 2
+            ),
+            .timedOut
+        )
```

Comment:
This approach of adding more assertions (examples) for a single unit of behavior is called _triangulation_.

One could argue that only having two examples doesn't really triangulate, but 🤷 : another example felt unnecessary.